### ISSUE TITLE
Kratos 0.7.* installed from a helm chart can't send e-mails

### ISSUE THREAD

**Issue by valentinyanakiev** [ory/k8s#336]
**Describe the bug**

Kratos has released version 0.7.6-alpha1 with significant updates but Helm chart uses 0.6.3-alpha1.
Manually specifying the version leads to exceptions in kratos courier.

**To Reproduce**

1. Install Ory Kratos from Helm chart with custom Kratos image defined, e.g.
image:
  repository: oryd/kratos
  tag: v0.7.6-alpha.1
  pullPolicy: IfNotPresent

2. Trigger login flow --> OK
3. Trigger logout flow --> OK
4. Trigger recovery / verification flow --> emails are not sent. Tested with mailslurper as part of the tech stack:
    courier:
      smtp:
        connection_uri: smtps://test:test@mailslurper-api:1025/?skip_ssl_verify=true
5. Checking the kratos courier logs show go exceptions.

**Expected behavior**

Emails should be sent. The same configuration works seamlessly with Kratos 0.6.3-alpha1.

**Comment by aeneasr** [comment]
This works fine on our env. Please make sure to read the changelog before upgrading: 
https://github.com/ory/kratos/blob/master/CHANGELOG.md

**Comment by valentinyanakiev** [comment]
> This works fine on our env. Please make sure to read the changelog before upgrading: https://github.com/ory/kratos/blob/master/CHANGELOG.md

Is your environment on k8s? How do you deploy it - I see the helm chart is outdated and has been since 0.7.* has been released - it deploys 0.6.3-alpha1 by default. Do you override it manually?

**Comment by aeneasr** [comment]
yes and yes - see also pinned issue: https://github.com/ory/k8s/issues/195
